FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/IFhirDataClient.cs

```diff
@@ -3,16 +3,21 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-using System;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Health.Fhir.Synapse.DataClient.Fhir;
+using Microsoft.Health.Fhir.Synapse.DataClient.Api;
 
 namespace Microsoft.Health.Fhir.Synapse.DataClient
 {
     public interface IFhirDataClient
     {
-        public Task<FhirElementBatchData> GetAsync(
+        /// <summary>
+        /// Returns a FHIR bundle which contains the matching search results.
+        /// </summary>
+        /// <param name="searchParameters">search parameters.</param>
+        /// <param name="cancellationToken">cancellation token.</param>
+        /// <returns>returned bundle.</returns>
+        public Task<string> SearchAsync(
```

Comment:
Can we directly return a ```JObject``` here? Cause we have already know the type of returned value.

Reply:
Yes, we can return a JObject. However, I want to return the raw search bundle directly to seperate post-process operations like serialization/extract continuation token to the pipeline. Seems it's fine to leave only API request logic to data client.